cms/templates/js/optional-content-editor.underscore

```diff
@@ -0,0 +1,5 @@
+<h3 class="modal-section-title">
+    <input type="checkbox" id="optional_content" name="optional_content"
+          class="input input-checkbox" />
+    <%- gettext('Mark as optional') %>
```

Comment:
We should add an explanation (perhaps with the `.field-message` class) to this field.

Reply:
modified in the other PR to look like this
![optional-editor-disabled](https://github.com/openedx/edx-platform/assets/19278837/d92610e6-25f5-416a-8899-b173da2a0c2d)